OpenRA.Game/Traits/Health.cs

```diff
@@ -188,6 +188,15 @@ public static DamageState GetDamageState(this Actor self)
 			return (health == null) ? DamageState.Undamaged : health.DamageState;
 		}
 
+		public static bool IsDamaged(this Actor self)
```

Comment:
This is introducing pretty much duplicate code/behavior. Currently everything uses the above method (`GetDamageState()`) to check if the actor is damaged. I see no reason why you can't use it as well.